Examples/UIExplorer/UIExplorerApp.android.js

```diff
@@ -39,11 +39,52 @@ const UIExplorerStateTitleMap = require('./UIExplorerStateTitleMap');
 
 var DRAWER_WIDTH_LEFT = 56;
 
+function PathActionMap(path: string): ?Object {
+  // Warning! Hacky parsing for example code. Use a library for this!
+  const exampleParts = path.split('/example/');
+  const exampleKey = exampleParts[1];
+  if (exampleKey) {
+    if (!UIExplorerList.Modules[exampleKey]) {
+      Alert.alert(`${exampleKey} example could not be found!`);
+      return null;
+    }
+    return UIExplorerActions.ExampleAction(exampleKey);
+  }
+  return null;
+}
+
+function URIActionMap(uri: ?string): ?Object {
+  // Warning! Hacky parsing for example code. Use a library for this!
+  if (!uri) {
+    return null;
+  }
+  const parts = uri.split('rnuiexplorer:/');
+  if (!parts[1]) {
+    return null;
+  }
+  const path = parts[1];
+  return PathActionMap(path);
+}
+
+type Props = {
+  openExampleName: String,
+};
+
 class UIExplorerApp extends React.Component {
+  constructor(props) {
+    super(props);
+    this._handleOpenExample = this._handleOpenExample.bind(this);
+    this.state = {openExampleName: props.openExampleName};
+  }
+
   componentWillMount() {
     BackAndroid.addEventListener('hardwareBackPress', this._handleBackButtonPress.bind(this));
   }
 
+  componentDidMount() {
+    setTimeout(this._handleOpenExample, 500);
```

Comment:
Can you have a note that explains why you had to make this hack, or at least your best guess, and warn people that this is hacky to do?

Also, can we rename `_handleOpenExample` to `_handleOpenInitialExample` to avoid confusion?